cea/optimization/optimization_main.py

```diff
@@ -63,8 +65,8 @@ def moo_optimization(locator, weather_file, gv, config):
 
     # optimize conversion systems
     print "CONVERSION AND STORAGE OPTIMIZATION"
-    master.evolutionary_algo_main(locator, building_names, extra_costs, extra_CO2, extra_primary_energy, solarFeat,
-                                  network_features, gv, config, prices, lca)
+    new_master.master_main(locator, building_names, extra_costs, extra_CO2, extra_primary_energy, solar_features,
```

Comment:
oh. i see. you have a function called `master_main` inside the module `master_main`. and... and what does that function do? i think the function might be able to do with a better name, what do you think?